osquery/filesystem/file_compression.cpp

```diff
@@ -33,7 +33,16 @@ DECLARE_uint32(carver_block_size);
 Status compress(const boost::filesystem::path& in,
                 const boost::filesystem::path& out) {
   PlatformFile inFile(in, PF_OPEN_EXISTING | PF_READ);
-  PlatformFile outFile(out, PF_CREATE_NEW | PF_WRITE);
+  if (!inFile.isValid()) {
+    return Status::failure("Could not open in file :  " + in.string() +
```

Comment:
nit, I feel the text should be `Could not open in file: `
- Remove the ` ` before `:`
- Remove the second ` ` after `:`